src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -133,6 +132,11 @@ def __enter__(self):
 
     def __exit__(self, exc_type, exc_val, exc_tb):
         self.close()
+        # Delete log files to prevent interference from the current run on the next execution.
```

Comment:
Please put this part into close function.

Reply:
fixed